lib/network_monitor.c

```diff
@@ -1015,14 +1015,36 @@ static void check_default_route(struct nm_addr_info *ai)
          */
         mptcpd_addr_get(ai);
 
-        if (l_netlink_send(ai->nm->rtnl,
-                           RTM_GETROUTE,
-                           0,
-                           &store,
-                           buf - (char *) &store,
-                           handle_rtm_getroute,
-                           ai,
-                           NULL) == 0) {
+	unsigned int result = 0;
```

Comment:
(We don't need to initialise `result` to 0 here, but I guess it doesn't hurt, and the compiler will optimise that. (same below))

Reply:
You are correct.  The initialization in this case is more of a defensive coding practice to make it easier to detect run-time issues where the variable could be used uninitialized.  That won't happen as implemented in this patch, but I still like to explicitly initialize in such cases on the off chance subsequent changes to the code make use of an uninitialized variable occur.

In any case, the latest change removes these `result` variables.